tests/test_file_download.py

```diff
@@ -84,6 +85,30 @@
 # One particular commit for DATASET_ID
 DATASET_SAMPLE_PY_FILE = "custom_squad.py"
 
+class TestDiskUsageWarning(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Test with 100MB expected file size
+        cls.expected_size = 100 * 1024 * 1024
+        cls.disk_usage_mock = Mock()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.disk_usage_mock.reset_mock()
```

Comment:
```suggestion
```

No need for a class-level mock. The `disk_usage_mock` value is already injected for you by the `@patch` decorator.